test/gateways/trexle_test.exs

```diff
@@ -54,6 +55,91 @@ defmodule Gringotts.Gateways.TrexleTest do
     description: "For our valued customer, Mr. Potter"
   ]
 
+  @invalid_amount_response ~s/{"error":"Payment failed","detail":"Amount must be at least 50 cents"}/
+  @invalid_card_response ~s/{"error":"Payment failed","detail":"Your card's expiration year is invalid."}/
+  @valid_request_response ~s/{"response":{"token":"charge_3e89c6f073606ac1efe62e76e22dd7885441dc72","success":true,"captured":false}}/
+
+  describe "core" do
```

Comment:
Why is this `describe` block called "core"?

Reply:
I'll change it and name it `Authorize integration point testing`